e2etest/zt_enumeration_filter_test.go

```diff
@@ -162,21 +162,6 @@ func TestFilter_RemoveFolder(t *testing.T) {
 	}, EAccountType.Standard(), EAccountType.Standard(), "")
 }
 
-func TestFilter_RemoveContainer(t *testing.T) {
```

Comment:
why is this test removed? unrelated but this seems like its not testing what the test title claims

Reply:
This was a duplicate of another test with a different name that didn't actually test the relevant functionality that was attempted to be tested. I could rewrite this test and add it back in if wanted, but I believe it relies upon account-wide testing functionality, which, makes it probably not something to pursue in this framework.

Reply:
(Could probably test it in the new E2E framework though!)

Reply:
Actually. I'm not even sure what this is supposed to test. I guess using a wildcard in container name? idk. The name doesn't provide much information, and the test, clearly not testing what it's trying to test, does not make much sense.